verifier/verifier.py

```diff
@@ -43,30 +43,36 @@ def __init__(self, code, response):
         self.code = code
         self.response = response
 
+
+####
+# SMTP RCPT error handlers
+####
 def handle_550(response):
     if any([keyword.encode() in response for keyword in blocked_keywords]):
         return "Blocked by mail server"
     return "Mailbox unavailable"
 
-def handle_551(response):
-    return "Mailbox has moved"
-
-def handle_421(response):
-    return "Try again later"
-
-def handle_450(response):
-    return "Mailbox busy"
-
-def handle_other(response):
-    return "Other Error"
 
+# https://www.greenend.org.uk/rjk/tech/smtpreplies.html#RCPT
+# https://sendgrid.com/blog/smtp-server-response-codes-explained/
 handle_error = {
+    # 250 and 251 are not errors
     550: handle_550,
-    551: handle_551,
-    450: handle_450,
-    421: handle_421,
+    551: lambda _: "Mailbox has moved",
+    552: lambda _: "Storage allocation exceeded",
+    553: lambda _: "Mailbox name not allowed / mailbox doesn't exist",
+    450: lambda _: "Mailbox unavailable/busy",
+    451: lambda _: "Local error processing, try again later",
+    452: lambda _: "Insufficient system storage",
+    500: lambda _: "Syntax error (command not recognised)",
+    501: lambda _: "Syntax error (in parameters or arguments)",
+    503: lambda _: "Bad sequence of commands",
+    521: lambda _: "This domain doesn't accept mail",
```

Comment:
I think this will mean `host_exist` = False

Reply:
Just 521, I assume?